traja/models/train.py

```diff
@@ -315,21 +328,21 @@ def fit(
                                 regressor_out, parameters
                             )
 
-                test_loss_forecasting /= len(test_loader.dataset)
+                test_loss_forecasting /= len(data_loader_to_evaluate.dataset)
                 print(
-                    f"====> Mean test set forecasting loss: {test_loss_forecasting:.4f}"
+                    f"====> Mean {current_set} set forecasting loss: {test_loss_forecasting:.4f}"
                 )
                 if self.classify:
                     accuracy = correct / total
                     if test_loss_classification != 0:
-                        test_loss_classification /= len(test_loader.dataset)
+                        test_loss_classification /= len(data_loader_to_evaluate.dataset)
                         print(
-                            f"====> Mean test set classifier loss: {test_loss_classification:.4f}; accuracy: {accuracy:.2f}"
+                            f"====> Mean {current_set} set classifier loss: {test_loss_classification:.4f}; accuracy: {accuracy:.2f}"
```

Comment:
You did not update the name of `test_loss_classification` to reflect its new usage.

Reply:
Done.